wdl2cwl/errors.py

```diff
@@ -46,7 +45,7 @@ def __exit__(
 
     def makeLead(self) -> str:
         """Caculate the error message prefix."""
-        pos: SourcePosition = cast(SourcePosition, self.item.pos)
+        pos: WDL.SourcePosition = cast(WDL.SourcePosition, self.item.pos)
```

Comment:
The import used breaks the tests on my IDE when trying to run a test individually. miniwdl exports `SourcePosition` in the main export, so I think it should be OK to use it instead, since it also fixes the IDE issue.